registration/templates/registration/login.html

```diff
@@ -1,7 +1,12 @@
 {% extends "base.html" %}
 {% load static %}
 {% block content %}
-<div id="login" style="background: #1B1B1B; padding-top: 40px;">
+<div id="login" style="background-color: black;">
```

Comment:
remove the following:  style="background-color: black;". it is in front of the picture.